.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "proseWrap": "always"
```

Comment:
I think we should use the `"preserve"` value for this option (which seems to be the default value) since sometimes wrapping is important and should be preserved (for example take a look at last lines of our `README.md` file).
It will still help with wrapping the long lines in markdown files, but it won't modify other intentionally made wrappings.